programs/perpetuals/src/state/pool.rs

```diff
@@ -1096,12 +1182,14 @@ mod test {
             swap_out: 100,
             stable_swap_in: 100,
             stable_swap_out: 100,
-            add_liquidity: 200,
-            remove_liquidity: 300,
+            add_liquidity: 0,
+            remove_liquidity: 0,
```

Comment:
set these prior test config base fees to 0 for better understanding of optimal fee structure